source/reference/metrics.txt

```diff
@@ -0,0 +1,270 @@
+.. _app-services-metrics:
+
+==============================
+App Services Metrics Reference
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+App Services provides a variety of metrics related to App Services and Device
+Sync. You can access your metrics data with the Admin API's
+:admin-api-endpoint:`getMetrics <tag/metrics/operation/adminGetMetrics>` endpoint.
+
+The content on this page is provided for your information. The exact metrics
+returned may be subject to change.
+
+.. Source derived from https://docs.google.com/spreadsheets/d/1cjQopjPUsEKde6TlqO38B0Ko6n4vTwl4tgJ-LHSbf7Y/edit#gid=1354315462
+
+Units
+-----
+
+App Services expresses each metric in one of the following units:
+
+- BYTES_PER_SECOND
+- MILLISECONDS
+- SCALAR
+- SCALAR_PER_SECOND
+
+A **scalar** is a singular value representing a magnitude or count. Its exact
+interpretation depends on the metric.
+
+A **scalar per second** represents a count over time ("rate"). The value is
+effectively the total count in the requested time window divided by the number
+of seconds in the window.
+
+Some scalar per second values are expressed as a 95th percentile. This means
+that the reading for the given metric is below the given value 95% of the time
+and above it the other 5% of the time.
```

Comment:
out of curiosity, do you know why this is the case?

seems weird to me that we're telling user that our metric is this good 5% or more of the time

Reply:
"Good" implies the metric is a value judgment and that higher is always better. It's neither. The 95th percentile scalar/sec type is used for things like response times where lower is actually better. It's just saying that MOST of the time, it's below X, but sometimes it's above X. I don't know the exact technical reason why it's presented this way.

Reply:
thanks for indulging my curiosity